### PR TITLE
Fixed #27699 -- Fixed parse_duration() with a negative number of seconds.

### PR DESCRIPTION
**Pull request by hardik1997**
https://code.djangoproject.com/ticket/27699

### CODE REVIEW

File: django/utils/dateparse.py [hunk 1]
```diff
@@ -122,8 +122,10 @@ def parse_duration(value):
         match = iso8601_duration_re.match(value)
     if match:
         kw = match.groupdict()
-        sign = -1 if kw.pop('sign', '+') == '-' else 1
+        """sign = -1 if kw.pop('sign', '+') == '-' else 1"""
```
Comment (timgraham) [inline django/utils/dateparse.py]: Remove?

File: tests/utils_tests/test_dateparse.py [hunk 2]
```diff
@@ -108,6 +108,12 @@ def test_fractions_of_seconds(self):
 
     def test_negative(self):
         self.assertEqual(parse_duration('-4 15:30'), timedelta(days=-4, minutes=15, seconds=30))
+		self.assertEqual(parse_duration('-8 10:15:30'), timedelta(days=-8, hours=10, minutes=15, seconds=30))
```
Comment (timgraham) [inline tests/utils_tests/test_dateparse.py]: Please use spaces rather than tabs for indentation.

File: django/utils/dateparse.py (hunk 1 above)
Comment (hardik1997) [inline django/utils/dateparse.py]: I used negative sign(-) in regex pattern. Thats why I removed it.

Comment (hardik1997) [inline django/utils/dateparse.py]: For case like (-5:5:5), generated answer was none. I fixed it.

Comment (timgraham) [inline django/utils/dateparse.py]: What I meant is that you've comment this line out with triple quotes.

**Comment by timgraham** [comment]
Updated in https://github.com/django/django/pull/7825.

**Pull request closed without merging**